Makefile

```diff
@@ -252,16 +252,25 @@ else
 buildx:
 endif
 
+BUILDX_COMMIT ?= v0.3.0
+export BUILDX_COMMIT
+
 bundles/buildx: BUILDX_DOCKERFILE ?= Dockerfile.buildx
-bundles/buildx: BUILDX_COMMIT ?= v0.3.0
 bundles/buildx: bundles ## build buildx CLI tool
 	# This intetionally is not using the `--output` flag from the docker CLI which is a buildkit option
 	# The idea here being that if buildx is being used, it's because buildkit is not supported natively
-	docker build -f $(BUILDX_DOCKERFILE) -t "moby-buildx:$(BUILDX_COMMIT)" \
+	docker build -f $(BUILDX_DOCKERFILE) -t "moby-buildx:$${BUILDX_COMMIT:-latest}" \
 		--build-arg BUILDX_COMMIT \
 		--build-arg BUILDX_REPO \
 		--build-arg GOOS=$$(if [ -n "$(GOOS)" ]; then echo $(GOOS); else go env GOHOSTOS || uname | awk '{print tolower($$0)}' || true; fi) \
 		--build-arg GOARCH=$$(if [ -n "$(GOARCH)" ]; then echo $(GOARCH); else go env GOHOSTARCH || true; fi) \
-		. && \
-		id=$$(docker create moby-buildx:$(BUILDX_COMMIT)); \
-		if [ -n "$${id}" ]; then docker cp $${id}:/usr/bin/buildx $@ && touch $@; docker rm -f $${id}; fi
+		.
+
+	id=$$(docker create moby-buildx:$${BUILDX_COMMIT:-latest}); \
+	if [ -n "$${id}" ]; then \
+		docker cp $${id}:/usr/bin/buildx $@; \
+		touch $@; \
```

Comment:
OK, `&&` is still needed before `touch`, since otherwise we'll end up having an empty file in case `docker cp` fails, and this is not what we want here.

Reply:
IOW,

`docker cp $${id}:/usr/bin/buildx $@ && touch $@; \`

otherwise all the same

Reply:
in fact it's OK as it is since with empty file, `$@ version` below will fail anyway so up to you

Reply:
Ah, hm, yeah, overlooked that; I'll fix it to be correct